src/meta/contributing.md

```diff
@@ -64,10 +64,8 @@ to check.
 
 ## Style Guide
 
-Articles should be written in
-[Markdown](https://daringfireball.net/projects/markdown/syntax)
-(not [AsciiDoc](http://asciidoctor.org/docs/asciidoc-syntax-quick-reference/)).
-Markdown is usually powerful enough and is a more common technology than AsciiDoc.
+Articles should be written in Markdown—
+specifically, the [CommonMark dialect][commonmark].
```

Comment:
```suggestion
specifically, the [`CommonMark` dialect][commonmark].
```

Reply:
I went the .spelling route on the basis that it's a trademark like CircleCI or Django, which are both in .spelling, instead of a literal reference to a code entity